_config.yml

```diff
@@ -1,4 +1,4 @@
-highlighter: rouge
```

Comment:
According to this post : https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0 Rouge is now the only supported highlighter.